pom.xml

```diff
@@ -78,7 +83,12 @@
 			<dependency>
 				<groupId>org.apache.hadoop</groupId>
 				<artifactId>hadoop-client</artifactId>
-				<version>1.0.3</version>
+				<version>2.6.0</version>
```

Comment:
And don't include the hadoop-client jar, include individual ones instead:

      h2Compile spec.external.mapredCore
      h2Compile spec.external.mapredClientCommon
      h2Compile spec.external.hadoopCommon